scripts/watch-client.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+let fs = require("q-io/fs");
+let path = require("path");
+let chokidar = require("chokidar");
+let colors = require("colors");
+
+let getListLocales = require("./common").getListLocales;
+let localizeClient = require("./localize-client");
+let env = require("../server/lib/environment");
+
+let root = process.cwd();
+let localeDir = path.join(root, env.get("L10N").locale_dest || "dist/locales");
+let dest = path.join(root, "client");
+let localeDirList;
+
+let log = {
+  args(color) {
+    return Array.prototype.map.call(this, arg => typeof arg === "string" ? colors[color](arg) : arg);
+  },
+  error() {
+    console.log.apply(console, arguments ? log.args.call(arguments, "red") : null);
+  },
+  info() {
+    console.log.apply(console, arguments ? log.args.call(arguments, "green") : null);
+  }
+};
+
+function deleteFile(relativeFilePath) {
+  let deleteLocalizedFile = localeDirList.map(localeDir => fs.remove(path.join(localeDir, relativeFilePath)));
```

Comment:
@cadecairos noted that it looked cleaner this way instead of stuffing all the code into one line.